include/quda.h

```diff
@@ -742,12 +766,6 @@ extern "C" {
         memory */
     QudaBoolean setup_minimize_memory;
 
-    /** Whether to compute the null vectors or reload them */
-    QudaComputeNullVector compute_null_vector;
```

Comment:
@weinbe2 with this gone I seem to hit an issue in tmLQCD (https://github.com/etmc/tmLQCD/pull/548) where QUDA attempts to load the null vectors from file by default. I do set the new per level `setup_type` to `QUDA_SETUP_NULL_VECTOR_INVERSE_ITERATIONS` and don't really know if I'm missing something else. Any pointer would be very helpful!

Reply:
Ah, I think I might have missed a major change in this PR: `setMultigridParam -> setWilsonMultigridParam`. I'll try that to see if it helps.

Reply:
For better or for worse, this is consistent with the behavior I put into `multigrid.cpp`:

```
if (strcmp(param.mg_global.vec_infile[param.level], "") != 0 && !refresh) {
  load[...]
} else {
  generate[...]
}
```

The reasoning behind it was if `vec_infile` is set, the user must want us to load near nulls, so you should check to make sure that value is the empty string.

That said... this behavior is worth revisiting. We could add a flag `override_load_by_default` or something like that, so it's not _only_ based on not setting `vec_infile`. I also didn't think about how that could be a pain in workflows that generate once, recreate the MG setup for one reason or another, then want to reload from there.

Let me know if you think keeping this default behavior (querying if the string isn't empty), but also adding an override flag, would be a good solution.

Reply:
Ah, I missed the fact that there are now per-level `vec_infile` and `vec_outfile` parameters. We actually did something really bad in the past:

```
  // set file i/o parameters
  strcpy(mg_param->vec_infile, "");
  strcpy(mg_param->vec_outfile, "");
```

which appears to be a bug in our interface dating back all the way to 1dec1db7a57e4fe00b14e04ba2088ca5039f8760 (June 2019)... Ouch :)


I think keeping this behaviour is historically consistent (also with the eigensolver).

Thanks a lot!